src/pages/developing/frameworks/vanilla.mdx

```diff
@@ -71,10 +72,8 @@ carbon-components/
 ├── html
 ├── css
 │   ├── carbon-components.css
-│   └── carbon-components.min.css
```

Comment:
I think the what's included section likely needs to be updated, same with the install step above.